pkg/sql/sqlbase/testutils.go

```diff
@@ -375,3 +379,169 @@ func TestingMakePrimaryIndexKey(desc *TableDescriptor, vals ...interface{}) (roa
 func TestingDatumTypeToColumnSemanticType(ptyp types.T) (ColumnType_SemanticType, error) {
 	return datumTypeToColumnSemanticType(ptyp)
 }
+
+// RandCreateTable creates a random CreateTable definition.
+func RandCreateTable(tableIdx int) *tree.CreateTable {
+	rand := rand.New(rand.NewSource(timeutil.Now().UnixNano()))
+	// columnDefs contains the list of Columns we'll add to our table.
+	columnDefs := make([]*tree.ColumnTableDef, randutil.RandIntInRange(rand, 1, 20))
```

Comment:
should the lower bound be `0`? i think we allow that and so should probably test it since it's an edge case.

Reply:
0 column tables are allowed? that's wild! i'll fix it, i guess.